src/database/dtos/customers.ts

```diff
@@ -1,8 +1,21 @@
 import { NeonDbError } from '@neondatabase/serverless'
-import { sql } from '@/database'
+import { DbResponse, sql } from '@/database'
 
-// TODO: Add logging
-// TODO: Add typing
+type Customer = {
+  id: number
+  email: string
+  first_name: string | null
+  last_name: string | null
+  password_hash: string | null
+  mobile_number: string | null
+  shopify_access_token: string | null
+  shopify_access_token_expires_at: string | null
+  accepts_marketing: boolean
+  shopify_cart_id: string | null
+  created_at: string
+  updated_at: string | null
+  deleted_at: string | null
+}
 
```

Comment:
type ListOfCustomerIds = Array<Pick<Customer, 'id'>>>

should we declare return type before the function ?